assets/syntaxes/02_Extra/INI.sublime-syntax

```diff
@@ -5,8 +5,8 @@ name: INI
 file_extensions:
   - ini
   - INI
-  - inf
-  - INF
+  - "inf"
```

Comment:
If I understood correctly, this `sublime-syntax` is manually converted from the `tmLanguage` in the INI submodule. Would it make sense to maintain this as a patch as opposed to directly editing the sublime-syntax?

Reply:
Thank you for code review! IMHO patches are a bit annoying and should be used sparingly, most appropriately when the file to be patched is not under our control. In this case the file happens to be versioned with us, so I think the most straightforward is to patch it directly, and then add a regression test so we detect if there is a regression. But if you strongly prefer a patch that is fine with me, I can change to that :)

Reply:
Your reasoning is sound, let's leave it how it is :)